pom.xml

```diff
@@ -316,7 +316,7 @@
       <dependency>
         <groupId>org.bdgenomics.bdg-formats</groupId>
         <artifactId>bdg-formats</artifactId>
-        <version>0.7.0</version>
+        <version>0.7.2-SNAPSHOT</version>
```

Comment:
[0.7.1 has propagated to Maven Central](http://search.maven.org/#artifactdetails%7Corg.bdgenomics.bdg-formats%7Cbdg-formats%7C0.7.1%7Cjar).